android/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
my suggestion would be to not check this in, might be easiest to give you my reasoning when we next chat